sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -5,7 +5,7 @@
 
 import com.azure.core.http.HttpPipelineBuilder;
 import com.azure.core.annotation.ServiceClientBuilder;
-import com.azure.core.http.policy.AddDatePolicy;
+import com.azure.core.http.policy.*;
```

Comment:
You'll need to update IntelliJ to not turn it into a star import after like 99 instead of what it has set by default.. 10?

Reply:
yes